File: dbflow/src/androidTest/java/com/raizlabs/android/dbflow/test/structure/join/Department.java
package com.raizlabs.android.dbflow.test.structure.join;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;
import com.raizlabs.android.dbflow.test.TestDatabase;

/**
 * Description: Taken from http://www.tutorialspoint.com/sqlite/sqlite_using_joins.htm
 */
@Table(database = TestDatabase.class)
public class Department extends BaseModel {

    @Column
    @PrimaryKey
    long id;

    @Column
    String dept;

    @Column
    long emp_id;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/structure/join/Department.kt
package com.raizlabs.android.dbflow.structure.join

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel
import com.raizlabs.android.dbflow.TestDatabase

/**
 * Description: Taken from http://www.tutorialspoint.com/sqlite/sqlite_using_joins.htm
 */
@Table(database = TestDatabase::class)
class Department : BaseModel() {

    @Column
    @PrimaryKey
    var id: Long = 0

    @Column
    var dept: String? = null

    @Column
    var emp_id: Long = 0
}
